File: sub-web-model/src/main/java/org/github/base/mapper/ICategoryMapper.java
package org.github.base.mapper;

import org.github.base.entity.CategoryEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 商品分类  Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2021-04-07
 */
@MyBatisMapper
public interface ICategoryMapper extends IMapper<CategoryEntity> {}

File: sub-mysql-course-service/src/main/java/org/github/mysql/course/base/mapper/ICategoryMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.course.base.mapper;

import org.github.mysql.course.base.entity.CategoryEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 分类 Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface ICategoryMapper : IMapper<CategoryEntity>
